MAINTAINERS.md

```diff
@@ -22,3 +22,34 @@ We are grateful to our former maintainers for their contributions to the Jaeger
 * [@objectiser](https://github.com/objectiser)
 * [@tiffon](https://github.com/tiffon)
 * [@vprithvi](https://github.com/vprithvi)
+
+### Maintainer Onboarding
+
+Upon approval, the following steps should be taken to onboard the new maintainer:
+
+*   **1. Update Project Documentation**
+    *   **`OWNERS` File:** Merge the PR to add the new maintainer to the `OWNERS` file(s) in the relevant Jaeger repositories.
```

Comment:
don't thin this is needed, as we only have a group there
```
$ cat .github/CODEOWNERS

* @jaegertracing/jaeger-maintainers
```
But *this* `MAINTAINERS.md` file needs to be updated.